mlrun/projects/operations.py

```diff
@@ -87,26 +87,26 @@ def run_function(
 
     example (use with function object)::
 
-        function = mlrun.import_function("hub://sklearn_classifier")
-        run1 = run_function(function, params={"data": url})
+        function = mlrun.import_function("hub://auto_trainer")
+        run1 = run_function(function, params={"label_columns": LABELS}, inputs={"dataset": DATA_URI})
```

Comment:
For the uri I would give a more random sample in the length of an actual uri.
```suggestion
        run1 = run_function(function, params={"label_columns":["target"]}, inputs={"dataset": "xxxxxxxxxx"})
```